plugins/polio/api/dashboards/supply_chain.py

```diff
@@ -172,7 +178,12 @@ class PreAlertDashboardViewSet(ModelViewSet):
     """
 
     http_method_names = ["get"]
-    permission_classes = [VaccineSupplyChainReadWritePerm]
+    permission_classes = [
+        lambda: VaccineStockManagementPermission(
+            non_admin_perm=permission.POLIO_VACCINE_SUPPLY_CHAIN_READ,
+            admin_perm=permission.POLIO_VACCINE_SUPPLY_CHAIN_WRITE,
```

Comment:
It's confusing that you put them in "non-admin, admin" order where in the constructor you did "admin, non-admin."

This should be consistent